### PR TITLE
fix bug of NaN left style

### PR DESCRIPTION
**Pull request by enbochen**
This PR solves the bug after upgrading to React Native 0.58.0

### CODE REVIEW

File: src/NavigatorNavigationBar.js [hunk 1]
```diff
@@ -140,6 +140,10 @@ class NavigatorNavigationBar extends React.Component {
       var component = this._components[componentName].get(this.props.navState.routeStack[index]);
       var props = this._getReusableProps(componentName, index);
       if (component && interpolate[componentName](props.style, amount)) {
+        if (isNaN(props.style.left)) {
+          props.style.left = undefiend;
```
Comment (sharplet) [inline src/NavigatorNavigationBar.js]:
```suggestion
          delete props.style.left;
```

**Review by sharplet — COMMENTED** [review]
Beside the typo, this fixed the issue I was seeing after upgrading to React Native 0.58.0 where Navigator transitions were completely broken. Thanks!

**Comment by sharplet** [comment]
FWIW, these transitions still appear to work on 0.59.0, but not on the latest version, 0.59.8 (instead of an animation, the new screen just pops in).

**Comment by SpencerLynn** [comment]
This likely will fix #37 

This fix is probably good, but should the styles also be fixed to not directly lead to this issue?

**Pull request closed without merging**